src/renderer/glescommon/gles.c

```diff
@@ -14,18 +14,37 @@
 #include "../gl33/gl33.h"
 
 void gles_init(RendererBackend *gles_backend, int major, int minor) {
-#ifdef TAISEI_BUILDCONF_HAVE_WINDOWS_ANGLE_FALLBACK
+#ifdef TAISEI_BUILDCONF_HAVE_ANGLE
+	// for loading ANGLE libraries
 	char *basepath = SDL_GetBasePath();
 	size_t basepath_len = strlen(basepath);
-	char buf[basepath_len + 32];
+	char buf[basepath_len + 256];
+
+#ifdef __APPLE__
```

Comment:
I think it's best to handle this in Meson rather than ifdeffing it. Have Meson export two new `TAISEI_BUILDCONF_` constants that contain paths to the GLESv2 and EGL libraries, relative to `SDL_GetBasePath()`. Then the C side can be simpler and OS-agnostic.

Note that it's also possible to configure a macOS build that does not produce an app bundle, by disabling the `macos_bundle` meson option. That should be either correctly handled, or made an explicit error with `install_angle`.

Reply:
Fixed relative install paths for macOS and implemented the `TAISEI_BUILDCONF_*` suggestion.